src/edu/ucsb/nceas/metacat/properties/AuthPropertiesDelegate.java

```diff
@@ -0,0 +1,134 @@
+/**
+ *    Purpose: A Class that handles Authentication-related properties for metacat configuration
+ */
+package edu.ucsb.nceas.metacat.properties;
+
+import edu.ucsb.nceas.utilities.FileUtil;
+import edu.ucsb.nceas.utilities.GeneralPropertyException;
+import edu.ucsb.nceas.utilities.MetaDataProperty;
+import edu.ucsb.nceas.utilities.PropertiesMetaData;
+import edu.ucsb.nceas.utilities.SortedProperties;
+
+import javax.xml.transform.TransformerException;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Set;
+import java.util.Vector;
+
+public class AuthPropertiesDelegate {
```

Comment:
Class javadocs  missing

Reply:
PropertyService, AuthPropertiesDelegate & BackupPropertiesDelegate addressed in commit 7a41ed9274d9c0e63125b03add225c38537dae91